providers/kafka_provider.ts

```diff
@@ -1,64 +1,29 @@
 import { ApplicationService, KafkaConfig } from '@adonisjs/core/types'
 
 import { Kafka } from '../src/index.ts'
-import { Producer } from '../src/producer.ts'
-
-import { HttpContext } from '@adonisjs/core/http'
-
-declare module '@adonisjs/core/http' {
-  export interface HttpContext {
-    kafka: {
-      producers: {
-        [key: string]: Producer
-      }
-    }
-  }
-}
```

Comment:
We've removed the kafka on HttpContext in favour of Producers & class based producers.

Reply:
This still needs a `node ace make:producer <producer> <topic>` that's like:

```typescript
import { Producer } from '#start/kafka'

export default class <topic|toCamalCase> {
  static topic = '<topic>'

  static async publish(message: string) {
    return Producer.send(<topic|toCamalCase>.topic, {
      value: string
    })
  }
}

```